{{cookiecutter.project_slug}}/clients/vue3/postcss.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   plugins: {
-    tailwindcss: {},
-    autoprefixer: {},
-  }
-}
\ No newline at end of file
+    tailwindcss: {
+      config: './tailwind.config.js',
+    },
+  },
+}
```

Comment:
**Info:** The Tailwind docs indicate to set up Tailwind with the previous syntax we had here.

However, this worked for Tailwind v2 with Vue3, but was not working for Tailwind v3 with Vue3.

Found a github discussion comment where a user indicated that this new syntax worked for them, so tried it out here and seems to have helped do the trick :eyes:

Referenced discussion comment can be found here:
https://github.com/tailwindlabs/tailwindcss/discussions/2854#discussioncomment-2047117